htcanalyze/htcanalyze.py

```diff
@@ -155,24 +156,30 @@ def htcondor_stdout(self, file: str) -> str:
             with open(file, "r") as output_content:
                 output_string += "".join(output_content.readlines())
         except NameError as err:
-            logging.exception(err)
-            rprint(f"[red]The smart_output_output method requires a "
-                   f"{self.ext_out} file as parameter[/red]")
+            handle_name_error(err, self.ext_out)
```

Comment:
This, much like `handle_filenotfound_error` is very specific, and only moves the code around. I'd keep it in here, since it's very short. It also explains the code here, because the error message should describe what exactly can be expected to go wrong when reading the code. Having to then search for the `handle_name_error` function just to get that is a bit cumbersome for no real reason.